roles/devscripts/templates/mpath.j2

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  name: 91-mpath-conf
+  labels:
+    machineconfiguration.openshift.io/role: {{ ocp_role }}
+    service: cinder
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - path: /etc/multipath.conf
+          overwrite: false
+          mode: 344
+          user:
+            name: root
+          group:
+            name: root
+          contents:
+            source: data:text/plain;charset=utf-8;base64,ZGVmYXVsdHMgew0KICB1c2VyX2ZyaWVuZGx5X25hbWVzICAgIG5vDQogIHJlY2hlY2tfd3dpZCAgICB5ZXMNCiAgc2tpcF9rcGFydHggICAgeWVzDQogIGZpbmRfbXVsdGlwYXRocyAgeWVzDQp9DQpibGFja2xpc3Qgew0KfQ==
```

Comment:
same question/remark: is this the content of the `multipath.conf` as exposed in the `files` - if so, would it be possible to generate that string instead?